tensorflow/tools/api/golden/v1/tensorflow.pbtxt

```diff
@@ -1032,6 +1032,14 @@ tf_module {
     name: "cross"
     argspec: "args=[\'a\', \'b\', \'name\'], varargs=None, keywords=None, defaults=[\'None\'], "
   }
+  member_method {
+    name: "cudnn_rnn_canonical_to_params_v2"
```

Comment:
These ops should be hidden in their apidef files to avoid them appearing here

Reply:
Do you mean I should delete both `cudnn_rnn_canonical_to_params_v2` and `cudnn_rnn_params_to_canonical_v2`? Or is there any attribute that I can set `hidden`?

Reply:
From API owners: 

> Need to add an api_def file in the python_api directory setting this op to hidden, like https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/api_def/python_api/api_def_CudnnRNN.pbtxt

Reply:
I think I have already added this `api_def_CudnnRNNCanonicalToParamsV2.pbtxt`.

Reply:
Could you help to redo [this](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/api/tests/README.txt)? I think it should eliminate these changes in the golden files. If not, please manually remove them.